src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-block">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label>
+            Surname:
+            <input type="text"
+            name="surname"
+            autocomplete="off"
+            >
```

Comment:
```suggestion
            <input 
              type="text"
              name="surname"
              autocomplete="off"
            >
```
start the first property on the new line, and don't forget about  2-space indentation (checklist p3) 

please fix it everywhere